packages/paste-website/src/pages/introduction/for-designers/design-guidelines/index.mdx

```diff
@@ -70,9 +70,9 @@ If you want to control libraries per file, go to the “Assets” tab on a file.
 also where you can drag in components to your file. Click the book icon in the upper
 corner. From the modal, you can turn libraries on and off.
 
-![Buttons to click in Figma to open the library selector](../../../assets/images/design-guidelines/figma-open-library-modal.png)
+![Buttons to click in Figma to open the library selector](../../../../assets/images/design-guidelines/figma-open-library-modal.png)
 
-![Modal in Figma where you can select the Paste components library to link to your project](../../../assets/images/design-guidelines/figma-link-paste-libraries.png)
+![Modal in Figma where you can select the Paste components library to link to your project](../../../../assets/images/design-guidelines/figma-link-paste-libraries.png)
```

Comment:
Is there a reason we are not using `Img` from `gatsby-img`?

Reply:
Yeah I can make a ticket!